Remove rpath from library, it points to build area which is not
ok

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: qwt-6.1.3/designer/designer.pro
===================================================================
--- qwt-6.1.3.orig/designer/designer.pro
+++ qwt-6.1.3/designer/designer.pro
@@ -83,7 +83,6 @@ contains(QWT_CONFIG, QwtDesigner) {
         # compile the path for finding the Qwt library
         # into the plugin. Not supported on Windows !
 
-        QMAKE_RPATHDIR *= $${QWT_INSTALL_LIBS}
         qwtAddLibrary($${QWT_OUT_ROOT}/lib, qwt)
 
         contains(QWT_CONFIG, QwtDll) {
